### PR TITLE
ENYO-1382 & ENYO-1383: Changes font size for ExpandableListItem and LabeledTextItem accordin…

### PR DESCRIPTION
**Pull request by JayCanuck**
…g the new guidelines

ENYO-1382: Expandable List Item font size needs to be 27px
ENYO-1383: Labeled Text Item: Label font size is smaller than text below

Enyo-DCO-1.1-Signed-off-by: Jason Robitaille <jason.robitaille@lge.com>

### CODE REVIEW

File: lib/ExpandablePicker/ExpandablePicker.less [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 /* Current Value */
 .moon-expandable-picker-current-value {
-	.moon-body-text;
+	.moon-item-body-text;
```
Comment (aarontam) [inline lib/ExpandablePicker/ExpandablePicker.less]: With this change, for 1080p, the `line-height` will change from `33px` to `36px` in a non-latin locale, and will change from `39px` to `39.5999984741211px` in other locales. I wanted to verify that this was intentional, thanks!

Comment (JayCanuck) [inline lib/ExpandablePicker/ExpandablePicker.less]: Updated to include line-height

Comment (aarontam) [inline lib/ExpandablePicker/ExpandablePicker.less]: Thanks! Will we need to add a rule like this for `moon-item-body-text` instead? https://github.com/enyojs/moonstone/blob/2.6.0-dev/css/moonstone-text.less#L179

The line-height is now `39px` in both non-latin and other locales.

**Comment by Djspaceg** [comment]
I think this set of changes is too complicated/elaborate for addressing these issues. I'm gonna take a look at re-evaluating both of these controls and do another PR. @JayCanuck are you ok with closing this PR?

**Pull request closed without merging**